script.js

```diff
@@ -0,0 +1,96 @@
+const body = document.querySelector('body');
+
+// menu mobile function
+
+bars.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+closing.addEventListener('click', () => {
+  menu.classList.toggle('active');
+});
+
+// this funcition is for mobile functionality
+links.forEach((link) => {
+  link.addEventListener('click', () => {
+    menu.classList.toggle('active');
+    body.classList.toggle('active');
+  });
+});
+
+// implimenting the section "our beautician artists"
+
+const beautician = [{
+  image: 'images/Esthetician .png',
+  name: 'Marleine Tshel',
+  description: 'Internatioal esthetician and over 8 years of experience.',
+  paragraph: 'Finished her esthetician studies and got her certificate in 2012. She provides services for your skin care needs.',
+}, {
+  image: 'images/Makeup artist .png',
+  name: 'Anabelle Slovy',
+  description: 'Part of the DMFA Makeup Academy Tutors at Harvard University.',
+  paragraph: 'Anabelle have always been passionate about art and one of the most exquisite expressions of makeup.',
+}, {
+  image: 'images/eshetician2.png',
+  name: 'Virgile Bual',
+  description: 'Internatioal esthetician and over 8 years of experience.',
+  paragraph: 'Finished her esthetician studies and got her certificate in 2012. She provides services for your skin care needs.',
+}, {
+  image: 'images/esthetician3.png',
+  name: 'Van Glor',
+  description: 'Part of the DMFA Makeup Academy Tutors at Harvard University.',
+  paragraph: 'Van have always been passionate about art and one of the most exquisite expressions of makeup.',
+}, {
+  image: 'images/make up artist.png',
+  name: 'Elizabeth Beth',
+  description: 'Part of the DMFA Makeup Academy Tutors at Harvard University.',
+  paragraph: 'Beth have always been passionate about art and one of the most exquisite expressions of makeup.',
+}, {
+  image: 'images/spa artist.png',
+  name: 'Lauraine Abdob',
+  description: 'Internatioal esthetician and over 8 years of experience.',
+  paragraph: 'Finished her esthetician studies and got her certificate in 2012. She provides services for your skin care needs.',
+}];
+
+const beauticianSec = document.querySelector('.artist-wrapper');
+beautician.forEach((beautician) => {
+  const artistCard = document.createElement('div');
+  artistCard.classList.add('artist-card');
+  artistCard.innerHTML = `
+    <div class="artist-description">
+      <div class="artist">
+      <div class="imgs">
+      <img class="artist-image" src="${beautician.image}" alt="artist"/>
+      <img class="back-img" src="images/squre photo.png" alt="background" />
+      </div>
+  <div class="art-desc">
+    <h4>${beautician.name}</h4>
+    <figcaption> <em> ${beautician.description}</em></figcaption>
+    <hr class="artist-underline">
+    <p class="artist-paragraph"> ${beautician.paragraph}</p>
+  </div> 
+  </div> 
+  </div> 
+  `;
+  beauticianSec.appendChild(artistCard);
+});
```

Comment:
- [ ] Great job with making requested changes but there are still a few things that could be improved on in order to have a more nice UI like the previous reviewer stressed.
  - [ ] I think going with his suggestion to make your `.artist-wrapper` container have a width of 90% i.e. 90% of the available viewport will make your speaker cards more responsive on smaller devices.
  - _Kindly check this [video extract](https://watch.screencastify.com/v/QswyMiHIp1HI28rCU8Pv) to see how far it has come and how better it can become_
  - [ ] [OPTIONAL] As you will notice that at width < 768px, each speaker's info is floating/flexing on a row. Fixing this will help improve user experience across smaller screens